frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/pages/employee/index.js

```diff
@@ -140,6 +140,7 @@ const App = ({ path }) => {
           <PrivateRoute path={`${path}/view-estimate`} component={() => <ViewEstimate {...{ path }} />} />
           <PrivateRoute path={`${path}/view-loi`} component={() => <ViewLOI {...{ path }} />} />
           <PrivateRoute path={`${path}/response`} component={() => <Response {...{ path }} />} />
+          <PrivateRoute path={`${path}/download`} component={()=> <HandleDownloadPdf {...{path}}/>}/>

```

Comment:
@burhanj-eGov In this case if we click on download it will change the url right?

Reply:
yes